frameworks/g-cloud-9/questions/services/resellingType.yml

```diff
@@ -17,12 +17,16 @@ type: radios
 options:
   - label: I’m not a reseller
     value: not_reseller
+    filter_label: Supplier is not a reseller
```

Comment:
We may want to store these without capitalisation on the first letter so that it looks more natural if we're going to describe searches as normal sentences (e.g. '1141 results found in Cloud hosting with staff security clearance that conforms to BS7858:2012').